### PR TITLE
Jamal

### PR DESCRIPTION
**Pull request by quantumew**
Basic YAML and JSON conversion tool.

### CODE REVIEW

**Comment by GoogilyBoogily** [comment]
Looks good for hacked together code :)

**Comment by quantumew** [comment]
Added alias 18f4661671edb84aa82560b693faee7a87af15e8

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Usage
+-----
+
+    ./jamal <action> [<input-file>]
+
+    Options:
+        -h --help       Show this message.
+
+    Arguments:
+        <action>        Conversion action. [yaml2json|json2yaml]
```
Comment (lnagle) [inline README.md]: This documentation doesn't have the `yamltojson` and `jsontoyaml` options. That said, I don't think they're really necessary and I'd prefer to have briefer`y2j` or `j2y` options.

File: jamal.go [hunk 2]
```diff
@@ -0,0 +1,132 @@
+//usr/bin/env go run $0 $@; exit;
```
Comment (lnagle) [inline jamal.go:1]: Why the double slash at the beginning of this?

File: jamal.go [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+	"os"
+)
+
+var logger = log.New(os.Stderr, "", 0)
+
+func main() {
+	doc := `Jamal
+
+        Command line interface for converting JSON to YAML and YAML to JSON.
+        Expects either a input file or data from stdin.
```
Comment (lnagle) [inline jamal.go]: an*

File: jamal.go (hunk 2 above)
Comment (quantumew) [reply jamal.go:1]: Unix automatically normalizes paths. Go treats // as a single line comment. That is how you do a Golang shebang.

**Comment by quantumew** [comment]
@lnagle93 and @GoogilyBoogily Thanks, added new aliases ->  2d281b7337202f541c59a194bc574f1d8f5a5d06

**Pull request merged**